src/main/java/dgen/dataset/Dataset.java

```diff
@@ -1,49 +1,36 @@
 package dgen.dataset;
 
-import dgen.column.Column;
 import dgen.tables.Table;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
+@Deprecated
```

Comment:
Is there any reason to keep this around? Shouldn't we just remove it? Or otherwise, let users use both the command line and the programmatic API to generate datasets.

Reply:
Removed in a507420a5460e77eef93fc4e7d6b1e0b559444b7